macOS/DuckDuckGo/AIChat/AIChatRemoteSettings.swift

```diff
@@ -81,7 +81,7 @@ struct AIChatRemoteSettings: AIChatRemoteSettingsProvider {
     }
 
     var isAIChatEnabled: Bool {
-        featureFlagger.isFeatureOn(.aiFeatures)
+        privacyConfigurationManager.privacyConfig.isEnabled(featureKey: .aiChat)
```

Comment:
I reverted this change to fix unit tests. As we agreed on the call, this flag (connected to privacy config) is a separate thing from the global AI toggle. I’m not removing .aiFeatures local flag as part of this change, but we should get rid of it too.

Then it turned out that this flag was actually in use – it decides about displaying AI Features pane in Settings (if disabled, AI Features is not shown at all). So I believe we should keep this property as is.